sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Infrastructure/StorageTestEnvironment.cs

```diff
@@ -54,6 +54,11 @@ public class StorageTestEnvironment: TestEnvironment
         ///
         public string StorageConnectionString => ActiveStorageAccount.Value.ConnectionString;
 
+        /// <summary>
+        ///   The storage account endpoint suffix of the cloud to use during Live tests.
+        /// </summary>
+        public string StorageEndpointSuffix => GetOptionalVariable("STORAGE_ENDPOINT_SUFFIX") ?? "core.windows.net";
```

Comment:
Can we create a storage account as part of ARM template deployment?

Reply:
We could, and that would work fine for CI purposes.    The test environment will use one if a connection string is provided and only create/destroy if none was specified.   Unless I'm not thinking of something, I think that would cover both scenarios well enough.

I've got some follow-up work in the next week or so to do for this in the area of the contributing guide and local templates.  I'll take a note to play with this and see about converting that over.

Reply:
Leaving this as a note for myself:   We also have the ability to detect an existing Event Hubs namespace by connection string.   See about having the Test Resources create both for CI runs.